src/discordcr/client.cr

```diff
@@ -72,7 +72,9 @@ module Discord
                    @shard : Gateway::ShardKey? = nil,
                    @large_threshold : Int32 = 100,
                    @compress : Bool = false,
-                   @properties : Gateway::IdentifyProperties = DEFAULT_PROPERTIES)
+                   @properties : Gateway::IdentifyProperties = DEFAULT_PROPERTIES,
+                   @logger = Logger.new(STDOUT))
+      @logger.progname = "discordcr"
```

Comment:
Should I build the below `@client_name` into this? I pass the `@logger` to our `WebSocket` instance which won't have `shard` information. On the other hand, `WebSocket` only has debug level logging at the moment, which you probably wouldn't use anyways for a production/sharded application.